package.json

```diff
@@ -52,7 +52,6 @@
     "@airbnb/config-prettier": "^3.1.0",
     "@airbnb/config-typescript": "^3.0.1",
     "@airbnb/nimbus": "^3.1.2",
-    "@superset-ui/commit-config": "^0.0.9",
```

Comment:
You can keep this. There is a cli in there for creating commit message that I use.

Reply:
Reverted the change!